<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-27T19:15:17 IST" name="Default suite" finished-at="2024-12-27T19:15:37 IST" duration-ms="20491">
    <groups>
    </groups>
    <test started-at="2024-12-27T19:15:17 IST" name="Default test" finished-at="2024-12-27T19:15:37 IST" duration-ms="20491">
      <class name="com.TestNGDemo.D15DDFLoginToOHRM">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:17 IST" name="beforeTest" finished-at="2024-12-27T19:15:23 IST" duration-ms="6057" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:23 IST" name="login" data-provider="getLoginData" finished-at="2024-12-27T19:15:25 IST" duration-ms="2157" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Nilesh]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nilesh123]]>
              </value>
            </param>
          </params>
          <exception class="graphql.AssertException">
            <message>
              <![CDATA[condition expected to be true]]>
            </message>
            <full-stacktrace>
              <![CDATA[graphql.AssertException: condition expected to be true
at graphql.Assert.assertTrue(Assert.java:85)
at com.TestNGDemo.D15DDFLoginToOHRM.login(D15DDFLoginToOHRM.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- graphql.AssertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:25 IST" name="logout" finished-at="2024-12-27T19:15:25 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:25 IST" name="login" data-provider="getLoginData" finished-at="2024-12-27T19:15:26 IST" duration-ms="1501" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[nilima]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nilima123]]>
              </value>
            </param>
          </params>
          <exception class="graphql.AssertException">
            <message>
              <![CDATA[condition expected to be true]]>
            </message>
            <full-stacktrace>
              <![CDATA[graphql.AssertException: condition expected to be true
at graphql.Assert.assertTrue(Assert.java:85)
at com.TestNGDemo.D15DDFLoginToOHRM.login(D15DDFLoginToOHRM.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- graphql.AssertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:26 IST" name="logout" finished-at="2024-12-27T19:15:26 IST" duration-ms="12" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:26 IST" name="login" data-provider="getLoginData" finished-at="2024-12-27T19:15:28 IST" duration-ms="1911" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:28 IST" name="logout" finished-at="2024-12-27T19:15:31 IST" duration-ms="2693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:31 IST" name="login" data-provider="getLoginData" finished-at="2024-12-27T19:15:33 IST" duration-ms="1577" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[anil]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[anil123]]>
              </value>
            </param>
          </params>
          <exception class="graphql.AssertException">
            <message>
              <![CDATA[condition expected to be true]]>
            </message>
            <full-stacktrace>
              <![CDATA[graphql.AssertException: condition expected to be true
at graphql.Assert.assertTrue(Assert.java:85)
at com.TestNGDemo.D15DDFLoginToOHRM.login(D15DDFLoginToOHRM.java:46)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- graphql.AssertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:33 IST" name="logout" finished-at="2024-12-27T19:15:33 IST" duration-ms="11" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:33 IST" name="login" data-provider="getLoginData" finished-at="2024-12-27T19:15:34 IST" duration-ms="1924" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:34 IST" name="logout" finished-at="2024-12-27T19:15:37 IST" duration-ms="2231" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:com.TestNGDemo.D15DDFLoginToOHRM@661972b0]" started-at="2024-12-27T19:15:37 IST" name="afterTest" finished-at="2024-12-27T19:15:37 IST" duration-ms="330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.TestNGDemo.D15DDFLoginToOHRM -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
